scripts/schemagen.go

```diff
@@ -0,0 +1,281 @@
+// Generates an initial version of a schema for a new resource type.
+//
+// This script draws heavily from https://github.com/radeksimko/terraform-gen,
+// but uses GCP's discovery API instead of the struct definition to generate
+// the schemas.
+//
+// This is not meant to be a definitive source of truth for resource schemas,
+// just a starting point. It has some notable deficiencies, such as:
+// 	* No way to differentiate between fields that are/are not updateable.
+// 	* Required/Optional/Computed are set based on keywords in the description.
+//
+// Usage example (from root dir):
```

Comment:
Can you add a note that users should use Google default credentials derived from `gcloud`?